### PR TITLE
rent create model and relations

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Create model Rent and relations with user and book models.

## Trellocard
https://trello.com/c/X75hubXV/19-crear-modelo-de-rent

## Screenshots
Captura de Pantalla 2019-10-07 a la(s) 13 09 30

Captura de Pantalla 2019-10-07 a la(s) 13 12 03

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 require 'faker'
 
 FactoryBot.define do
+  factory :rent do
+    association :user
```
Comment (aleochoam) [inline spec/factories.rb]:
```suggestion
    user
```

Comment (aleochoam) [reply spec/factories.rb]: It's the  same if you want

File: spec/models/rent_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { build(:rent) }
+
+  it { is_expected.to be_valid }
+  it { is_expected.to validate_presence_of(:user) }
+  it { is_expected.to validate_presence_of(:book) }
+  it { is_expected.to validate_presence_of(:start_date) }
+  it { is_expected.to validate_presence_of(:end_date) }
```
Comment (aleochoam) [inline spec/models/rent_spec.rb:10]: Let's also test the relationships

File: spec/factories.rb (hunk 1 above)
Comment (franciscoabalan) [reply spec/factories.rb]: Done

File: spec/models/rent_spec.rb (hunk 2 above)
Comment (franciscoabalan) [reply spec/models/rent_spec.rb:10]: DONE

**Review by aleochoam — APPROVED** [review]
Perfect!

**Review by joredudiaz — APPROVED** [review]
👍

**Pull request merged**